packages/vm/src/runBlock.ts

```diff
@@ -160,10 +160,10 @@ export async function runBlock(vm: VM, opts: RunBlockOpts): Promise<RunBlockResu
     // Populate the execution witness
     stateManager.initVerkleExecutionWitness!(block.header.number, block.executionWitness)
 
-    if (stateManager instanceof StatelessVerkleStateManager) {
+    if ('verifyPostState' in stateManager) {
```

Comment:
The line was previously very descriptive `stateManager instanceof StatelessVerkleStateManager` such that it is clear that `stateManager` is actually a `StatelessVerkleStateManager`. Could you add a comment here? `Check if stateManager is of type StatelessVerkleStateManager` for instance